### PR TITLE
Database

### PR DESCRIPTION
**Pull request by hamir**
Cleans up the interface to DB - now you just do an "adduser" as a dict with each key representing column name and value representing the corresponding data for that column.

---
This change is [Reviewable](https://reviewable.io/reviews/hamir/islam-buddy/18)

### CODE REVIEW

File: db.py [hunk 1]
```diff
@@ -5,25 +5,25 @@
...
+        id=google_id, 
+        user_info=data.get('user_info'),
+        city=data.get('city'))
     user.put()
 
-  def GetUserInfo(self, google_id):
+  def GetUser(self, google_id):
     """Gets user info for the specified google ID."""
     key = ndb.Key(User, google_id).get()
-    if key:
```
Comment (a27med) [inline db.py]: Why did you remove this check?

File: app.py [hunk 2]
```diff
@@ -28,25 +28,26 @@ def home():
   return 'Welcome to the Islam Buddy API!'
 
 
-"""
```
Comment (a27med) [inline app.py]: Just to confirm, this is still a debug endpoint but we no longer add using the add command correct? If so, when we push to prod, lets not expose it anyways?

Comment (hamir) [reply app.py]: yeah i'm still figuring out a good way to manage this, but for now it's helpful for debugging

**Comment by hamir** [comment]
Thanks!
